dune/grid/cpgrid/Geometry.hpp

```diff
@@ -398,8 +398,10 @@ namespace Dune
             /// This method is meaningless for singular geometries.
             GlobalCoordinate corner(int /* cor */) const
             {
-                OPM_THROW(std::runtime_error, "Meaningless call to cpgrid::Geometry::corner(int): "
-                      "singular geometry has no corners.");
+                // Meaningless call to cpgrid::Geometry::corner(int): 
+                //"singular geometry has no corners.
+                // But the DUNE tests assume at least one corner.
+                return GlobalCoordinate( 0.0 );
```

Comment:
In general I am not very happy about meaningless calls being allowed, so I would prefer that the tests were rewritten. I do realize we cannot do that, in particular for existing stable dune versions, so I'll accept this. However, could we at least return points that are likely to trigger other problems, such as `return GlobalCoordinate(-1e100);` or similar?